code/_onclick/mindUI/soulblade.dm

```diff
@@ -0,0 +1,73 @@
+
+////////////////////////////////////////////////////////////////////
+//																  //
+//					   SOULBLADE (BLOOD GAUGE)					  //
+//																  //
+////////////////////////////////////////////////////////////////////
+
+/datum/mind_ui/soulblade
+	uniqueID = "Soulblade"
+	x = "LEFT"
```

Comment:
I am aware we haven't required this before, but this should probably just be a define.

Reply:
screen_loc are strings though, ones that are composed of several parts either. I think it's best to keep the elements that compose it as explicit as possible. Or do you think we should replace every "LEFT:16, CENTER+3:8" with "[LEFT]:16, [CENTER]+3:8" ? I don't think it's sensible.

Reply:
should I rename the var "screen_loc_x_base" so its purpose is more clear?

Reply:
And lastly for the love of god lets not go back to shit like: 
![image](https://user-images.githubusercontent.com/7573912/127979496-edb1178d-b91f-4078-968c-1bc083372b29.png)

defines used only once, defined in another file which makes you needlessly jump between files.

Reply:
No, it's pointless if they are cobbled like this, Defines are supposed to preempt typos but it won't help here.